core/templates/dev/head/components/forms/FormBuilder.js

```diff
@@ -521,6 +523,12 @@ oppia.directive('ckEditorRte', [
             .css('height', '26px')
             .css('width', '26px');
           ck.setData(wrapComponents(ngModel.$viewValue));
+
+          if (!canUseFs) {
+            $('.cke_button__oppiaimage').attr('onclick', '').unbind('click');
```

Comment:
Rather than specifically disabling the image component, you should check whether `componentDefn.requiresFs` is true to determine whether a component requires using the filesystem (if it does, and `canUseFs` is false, then you disable. Please refer to the way textAngular handles this.

Also, please check if there is a cleaner way to disable a CKEditor toolbar button--manually adjusting the CSS does not seem correct. A quick search led me [here](https://stackoverflow.com/a/23521552), maybe that is helpful

Reply:
Fixed this, PTAL!